src/extension.ts

```diff
@@ -50,8 +57,10 @@ export function activate(context: ExtensionContext) {
     /**
      * Delegate notifications from Java JDT LS to the MicroProfile LS
      */
-    bindNotification('microprofile/propertiesChanged');
-
+    context.subscriptions.push(commands.registerCommand('microprofile/propertiesChanged', (event: MicroProfilePropertiesChangeEvent) => {
+      languageClient.sendNotification('microprofile/propertiesChanged', event);
+      yamlSchemaCache.then(cache => cache.evict(event));
```

Comment:
Please check if cache is not undefined first `if (cache) cache.evict(event)`

Reply:
done